README.md

```diff
@@ -196,6 +196,10 @@ directory. The instructions note:
    >Zip the directory content, not the directory. The contents of the Zip file
    >are available as the current working directory of the Lambda function.
 
+Please note that if your pip version <10 then you might have issues building 
```

Comment:
We can get rid of this now that deploy_tools is updated and shouldn't care which version of pip you are using.

Reply:
@curranjm this seems to still be an issue for some, such as @SKalmane's arcgis acting up.

Reply:
@Joshhw, those problems are local to his environment and unrelated to the pip version. Pip version only mattered because we were using a deprecated function from v9 before, but I have removed all calls to that function. You can safely remove this.

Reply:
@Joshhw, those problems are local to his environment and unrelated to the pip version. Pip version only mattered because we were using a deprecated function from v9 before, but I have removed all calls to that function. You can safely remove this.